src/constants/navItems.js

```diff
@@ -0,0 +1,10 @@
+export default [
```

Comment:
Since it's not used anywhere else, I would suggest to not make a separate file for this.